x-pack/test/functional/apps/canvas/expression.ts

```diff
@@ -14,14 +14,16 @@ export default function canvasExpressionTest({ getService, getPageObjects }: Ftr
   const retry = getService('retry');
   const PageObjects = getPageObjects(['canvas', 'common']);
   const find = getService('find');
-  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
 
   describe('expression editor', function () {
     // there is an issue with FF not properly clicking on workpad elements
     this.tags('skipFirefox');
 
     before(async () => {
-      await esArchiver.load('x-pack/test/functional/es_archives/canvas/default');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/canvas/default'
+      );
       // load test workpad
```

Comment:
Should we be unloading this as well?  Seems like other tests do.

Reply:
After adding the unload, there are no extra saved objects, only the `Advanced Settings [VERSION NUMBER]` is still present.  At least in this test.